vitrina/datasets/models.py

```diff
@@ -124,7 +124,11 @@ def __str__(self):
         return self.title
 
     def get_absolute_url(self):
-        return reverse('dataset-detail', kwargs={'slug': self.slug})
+        return reverse('dataset-detail', kwargs={
+            'organization_kind': self.organization.kind if self.organization else None,
+            'organization_slug': self.organization.slug if self.organization else None,
```

Comment:
```suggestion
            'organization_kind': self.organization.kind,
            'organization_slug': self.organization.slug,
```

`Dataset.oranization` should be required property, means should be always set. And if for some reason it is not, we don't want to silence it.

Reply:
Removed these attributes from url based on #147